test/NonSilo.Tests/SchedulerTests/OrleansTaskSchedulerAdvancedTests_Set2.cs

```diff
@@ -685,7 +685,8 @@ public void ActivationSched_SubTaskExecutionSequencing()
                 SiloAddress = SiloAddressUtils.NewLocalSiloAddress(23)
             };
             var grain = NonReentrentStressGrainWithoutState.Create(grainId, new GrainRuntime(Options.Create(new ClusterOptions()), silo, null, null, null, null, null, NullLoggerFactory.Instance));
-            await grain.OnActivateAsync();
+
+            await Task.Factory.StartNew(() => grain.OnActivateAsync(), CancellationToken.None, TaskCreationOptions.None, scheduler).Unwrap();
```

Comment:
Previously we were violating scheduling rules here (OnActivateAsync must be run on the activation's scheduling context)